jiant/tasks/tasks.py

```diff
@@ -3731,3 +3732,132 @@ def get_metrics(self, reset=False):
         """Get metrics specific to the task"""
         acc = self.scorer1.get_metric(reset)
         return {"accuracy": acc}
+
+
+@register_task("wikipedia_corpus_sop", rel_path="wikipedia_corpus_small")
+class SentenceOrderTask(Task):
+    """ Task class for Sentence Order Prediction (SOP). See the ALBERT paper for details on SOP:
+        https://arxiv.org/abs/1909.11942.
+        We are currently using an unpreprocessed version of the Wikipedia corpus
+        (more specifically, the Wikidump data) that consists of 5% of the data. You can generate
```

Comment:
Question: Where do we specify the specific dump that we're downloading? Does the NVidia code always download the latest dump? If so, that's a problem for reproducibility.

Reply:
Seems like it downloads from here: https://dumps.wikimedia.org/enwiki/latest/enwiki-latest-pages-articles.xml.bz2. @phu-pmh  do you know if that's static or if it's overwritten with the latest dump?

Reply:
Seems like it does keep changing. Phu is working on a modified script that reads a dump from a particular date.

Reply:
Yes, seems like NVIDIA code always download the latest version. The dumb version we used is `https://dumps.wikimedia.org/enwiki/20200301/`. We can edit their code to download the same version as ours but it seems that there is copyright issues. 
According to @pyeres "NVIDIA files are Apache 2.0 licensed, and (depending on whether we’re redistributing or modifying+redistributing, we’d need to add some notices"

Reply:
@sleepinyourhat

Reply:
If the script is easy to edit, one option would be to give instructions here for how to edit it to specify a version.

Reply:
I'll update this in MLM branch's preprocessing script `jiant/scripts/mlm`.

Reply:
@phu-pmh  would be great if you updated this in the MLM branch.